libsrc/stdio/fread.c

```diff
@@ -4,7 +4,7 @@
 #define STDIO_ASM
 #include <stdio.h>
 
-#ifdef __8080
+#if CPU_8080 || CPU_8085
```

Comment:
__8080 || __8085 is preferred I think

Reply:
resolved with https://github.com/z88dk/z88dk/pull/2560/commits/4442b58a6164bfc4e250735af40ea4b625895b87